Solution converted to use new CSPROJ project type / targets .NET Standard 1.6

agecas opened the pull request:
Solution converted to use new CSPROJ project type / project retargeted to .NET Standard 1.6 / nugets updated to their latest versions.

nblumhardt commented:
Hi Aurimas, thanks for the PR.

Looks like the build failed; this needs an updated Build.ps1 and appveyor.yml file. You can find examples in, e.g.: https://github.com/serilog/serilog-formatting-compact-reader

Cheers!

nblumhardt commented on src/Serilog.Sinks.Logentries/Serilog.Sinks.Logentries.csproj:
```diff
@@ -0,0 +1,27 @@
+    <Description>Serilog event sink that writes to the Logentries.com service.</Description>
+    <VersionPrefix>3.0.0</VersionPrefix>
+    <Authors>Serilog Contributors;Michiel van Oudheusden</Authors>
+    <TargetFramework>netstandard1.6</TargetFramework>
```
The package should target .NET 4.5 and .NET Standard 1.3.

agecas commented:
Thanks for your reply and apologies for not checking the build. I have one
question however, is there are reason we are not targeting
451/452/46/461/462? Since this has potential to be used in any .NET
version, i doubt mainstream of development is happening at net45 at the
moment.

Thanks,
kind regards,
Aurimas Gecas

On Wed, Jun 7, 2017 at 10:39 PM, Nicholas Blumhardt <
notifications@github.com> wrote:

> *@nblumhardt* requested changes on this pull request.
>
> LGTM, with build/targeting issues covered.
> ------------------------------
>
> In src/Serilog.Sinks.Logentries/Serilog.Sinks.Logentries.csproj
> <https://github.com/serilog/serilog-sinks-logentries/pull/16#discussion_r120751424>
> :
>
> > @@ -0,0 +1,27 @@
> +﻿<Project Sdk="Microsoft.NET.Sdk">
> +
> +  <PropertyGroup>
> +    <Description>Serilog event sink that writes to the Logentries.com service.</Description>
> +    <VersionPrefix>3.0.0</VersionPrefix>
> +    <Authors>Serilog Contributors;Michiel van Oudheusden</Authors>
> +    <TargetFramework>netstandard1.6</TargetFramework>
>
> The package should target .NET 4.5 and .NET Standard 1.3.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/serilog/serilog-sinks-logentries/pull/16#pullrequestreview-42741823>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQC99YsWOW5k8tosgqjQrAJCFn5Q3gdhks5sBxh1gaJpZM4Ny3pz>
> .
>



-- 
Thank you,
Kind regards,
Aurimas Gecas

agecas commented:
hey once again mate,

So I have retargeted the project to .netstandard1.3;net45 and amended the
build script, created AppVeyor account and I can see my dev branch is
passing now, for my purposes I have also removed the nuget publishing,
since it was failing the build, due to the fact that the key is encrypted
on your account, not mine, not sure how to deal with that. I guess my
previous pull request is still valid. Please let me know if there is
anything else you need me to do in order to make this merge to happen.

Thanks,
Kind regards,
Aurimas Gecas

On Wed, Jun 7, 2017 at 10:39 PM, Nicholas Blumhardt <
notifications@github.com> wrote:

> *@nblumhardt* requested changes on this pull request.
>
> LGTM, with build/targeting issues covered.
> ------------------------------
>
> In src/Serilog.Sinks.Logentries/Serilog.Sinks.Logentries.csproj
> <https://github.com/serilog/serilog-sinks-logentries/pull/16#discussion_r120751424>
> :
>
> > @@ -0,0 +1,27 @@
> +﻿<Project Sdk="Microsoft.NET.Sdk">
> +
> +  <PropertyGroup>
> +    <Description>Serilog event sink that writes to the Logentries.com service.</Description>
> +    <VersionPrefix>3.0.0</VersionPrefix>
> +    <Authors>Serilog Contributors;Michiel van Oudheusden</Authors>
> +    <TargetFramework>netstandard1.6</TargetFramework>
>
> The package should target .NET 4.5 and .NET Standard 1.3.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/serilog/serilog-sinks-logentries/pull/16#pullrequestreview-42741823>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQC99YsWOW5k8tosgqjQrAJCFn5Q3gdhks5sBxh1gaJpZM4Ny3pz>
> .
>



-- 
Thank you,
Kind regards,
Aurimas Gecas

nblumhardt commented:
Hi @agecas - the NuGet step won't run during PR builds, so there shouldn't be any failure from it unless our build is misconfigured. It's needed as part of the build so that packages make it onto the NuGet feed. If you re-enable it and it fails, I can take a look at the error output 👍

Also needs to keep the "deploy on dev/master" setting.

HTH!
Nick

agecas commented:
Hi,

Ok so I have added back the nuget provider with the original API key,
however that does cause a failure:


[image: Inline image 1]


Which makes sense since it is a FORK and the key is from your account, so i
guess that is not a problem. The other thing is I have added GitHub
provider as per the example you pointed me to (
https://github.com/serilog/serilog-formatting-compact-reader/blob/dev/appveyor.yml),
however I have not updated the auth-token, since I am not sure what it
should be, so you need to fill this in yourself or remove the GitHub
provider if its not required.

Sorry for taking so long to fix such a simple issue, please let me know if
there is anything else i can do or if something is unclear.

Thank you,
Kind regards,
Aurimas Gecas

On Fri, Jun 9, 2017 at 5:31 AM, Nicholas Blumhardt <notifications@github.com
> wrote:

> Hi @agecas <https://github.com/agecas> - the NuGet step won't run during
> PR builds, so there shouldn't be any failure from it unless our build is
> misconfigured. It's needed as part of the build so that packages make it
> onto the NuGet feed. If you re-enable it and it fails, I can take a look at
> the error output 👍
>
> Also needs to keep the "deploy on dev/master" setting.
>
> HTH!
> Nick
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/serilog/serilog-sinks-logentries/pull/16#issuecomment-307292639>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQC99SVAEogJYGyV6tlLPVbj4PR79qA8ks5sCMqagaJpZM4Ny3pz>
> .
>



-- 
Thank you,
Kind regards,
Aurimas Gecas

nblumhardt commented:
Thank you, looks good 👍
